masonry/src/properties/checkmark.rs

```diff
@@ -0,0 +1,104 @@
+// Copyright 2025 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use std::any::TypeId;
+
+use crate::core::{Property, UpdateCtx};
+use crate::peniko::color::{AlphaColor, Srgb};
+
+/// The color of a checkbox's "check" icon.
+#[expect(missing_docs, reason = "obvious")]
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub struct CheckmarkColor {
```

Comment:
`EnabledCheckmarkColor` for consistency with `DisabledCheckmarkColor`? (Assuming properties always have a value and they cannot be unset, is that true?)

Reply:
Properties can be unset.

But also, the different `DisabledFoobar/HoveredFoobar/ActiveFoobar` properties are kind of a hack anyway, we're bound to get rid of them soon, so don't want to worry about them too much.

Reply:
But if they're unset then the default value for them is used? That is they always have a value?

Reply:
Yeah, right now there's no cascading between these different properties.

Reply:
I've added a TODO comment about this, but I'm still calling it `CheckmarkColor` for consistency with other properties. If we do decide to rename all baseline properties, I'd rather we do it in a dedicated PR.